### ISSUE TITLE
Progress has no "role" associated with it

### ISSUE THREAD

**Issue by rrichgithubwork** [KaterinaLupacheva/react-progress-bar#50]
Hi there, after doing an accessibility report your plugin was flagged for not having a associated **role**:
Here is what the report had:

Form control with"progressbar" role has no associated label.

If the control is a labelable element, add a for attribute to a visible label with the id of 
this control (e.g. <label for="controlID">);

Or, if the control is a labelable element, put the control inside a visible <label> element;

Or, add an aria-labelledby attribute to the control. It must point to visible text on the 
page that is meaningful as a label;

Or, if the design cannot have a visible label, provide a label using the aria-label 
attribute (e.g. aria-label="Activities");

Or, if the design cannot have a visible label, use the title attribute to provide a label

Suggested fix:
<div aria-valuetext="100%"aria-valuemax="100" aria-valuemin="0" aria-valuenow="100" **role="progressbar"** dir="ltr"class="ProgressBar">
